packages/schematics/.vscode/launch.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
AS this seems helpful is creating one small issue every time that someone wants to change or store some local settings for the project will trigger git change that can't be committed by mistake or not suitable for everyone. 

We have the `.vscode` folder already tracked that are creating a lot of thous issues mentioned above.

So I'm ok to discuss it ... but I think that editor specific configuration doesn't have to be part of the repository. 

A better solution will be to have a section into the documentation for improving the development experience for people but not force them to use it.

Reply:
I see what's the issue with tracking these configuration files. I'll document the setup instead.